compiler/damlc/lib/DA/Cli/Damlc/Test.hs

```diff
@@ -59,15 +60,22 @@ testRun h inFiles lfVersion color mbJUnitOutput  = do
     deps <- runActionSync h $ mapM getDependencies inFiles
     let files = nubOrd $ concat $ inFiles : catMaybes deps
 
-    results <- runActionSync h $
+    results <- runActionSync h $ do
+        dalfs <- Shake.forP files $
```

Comment:
Note that there is a slight inconsistency. We run scripts only in the main dalf while we test coverage against all dalfs. I think the choice you’ve taken here is sensible since it handles the case where you separate tests from templates but maybe it’s worth either adding a flag that runs tests in all dalfs (seems sensible anyway) or to limit coverage to only templates from certain dalfs. I’d leave it out of this PR though.

Reply:
I'll add it to the next PR.